build/automation/var/profile/dev.mk

```diff
@@ -5,10 +5,14 @@
 
 LOG_LEVEL:= DEBUG
 
-DB_SERVER_NAME := uec-core-dos-regression-db-12
-DB_REPLICA_SERVER_NAME := uec-core-dos-regression-db-12-replica-di
+R53_DB_SERVER_NAME := uec-core-dos-integration-dos-primary-rw.dos-db-rds
```

Comment:
If you rename to DB_SERVER then you can remove the exports for this in `populate-deployment-variables`